installer/roles/local_docker/tasks/main.yml

```diff
@@ -35,18 +45,24 @@
       -v {{ postgres_data_dir }}/10/data:/var/lib/postgresql/10/data \
       -e PGUSER={{ pg_username }} -e POSTGRES_INITDB_ARGS="-U {{ pg_username }}" \
       tianon/postgres-upgrade:9.6-to-10 --username={{ pg_username }}
-  when: upgrade_postgres | bool
+  when:
+    - pg_hostname is not defined
+    - upgrade_postgres | bool
 
 - name: Copy old pg_hba.conf
   copy:
     src: "{{ postgres_data_dir + '/pgdata/pg_hba.conf' }}"
     dest: "{{ postgres_data_dir + '/10/data/' }}"
-  when: upgrade_postgres | bool
+  when:   when:
+    - pg_hostname is not defined
+    - upgrade_postgres | boolupgrade_postgres | bool
```

Comment:
Is the `boolupgrade_postgres` part a typo?

Reply:
Yes, that is a type, my bad

Reply:
I have fixed the typo